sql_user_check_dynamodb/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+
+if ! command -v "aws" &> /dev/null
+then
+    echo "Command aws could not be found, please install it."
+    exit 1
+fi
+
+aws configure
```

Comment:
aws configure sets the credentials permanently. This is dangerous because the credentials will be kept in the machine. Instead use export to set the variables and unset to unset after script is complete

Reply:
done, prompting user now for credentials

Reply:
done.